modules/javafx.graphics/src/main/native-glass/gtk/glass_window.cpp

```diff
@@ -352,6 +345,14 @@ void WindowContextBase::process_mouse_motion(GdkEventMotion* event) {
             com_sun_glass_events_KeyEvent_MODIFIER_BUTTON_FORWARD);
     jint button = com_sun_glass_events_MouseEvent_BUTTON_NONE;
 
+    if (isDrag) {
+        // Upper layers expects from us Windows behavior:
+        // all mouse events should be delivered to window where drag begins
+        // and no exit/enter event should be reported during this drag.
+        // We can grab mouse pointer for these needs.
+        grab_mouse_drag_focus();
```

Comment:
Now that this code has moved here, it will be called every time the mouse moves during a drag operation. It should only be called once when the drag first starts.

Reply:
Fixed